rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestLogGroupBuilder.xtend

```diff
@@ -45,10 +46,21 @@ class TestLogGroupBuilder {
 
 	def void updateCurrentEntryAfterLogGroup(String logLine, List<TestLogGroup> result) {
 		if (logLine.contains("[TE-Test:")) {
+			if (logLine.contains("Running test") &&
+				lastLine.contains(" ****************************************************")) {
```

Comment:
We should put '[TE-Test:', 'Running test' and ' ***' in constants, too, right?
We want these constants to reside within a mutually used "component" in the long run ...